org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "opent2t-translator-com-wink-sensorpod",
+    "version": "0.2.2",
```

Comment:
>"version": "0.2.2"

This version number is just copied directly from the wink light package. Is there a canonical version number that we're using for new translators?